src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1553,7 +1553,6 @@ def __call__(self, hidden_states):
         return self.LayerNorm(hidden_states)
 
 
-# Copied from transformers.models.bert.modeling_flax_bert.FlaxBertLMPredictionHead with Bert->BigBird
```

Comment:
Hey @Bearnardd! Great, thanks! We can do a sneaky trick here where we do:
```
# Copied from transformers.models.bert.modeling_flax_bert.FlaxBertLMPredictionHead with Bert->BigBird, np->jnp
```
to copy the class but change the type annotation.

Otherwise this PR is ready for a final review! Super work on getting it finished - it was quite a tricky PR handled very well with some new JAX PRNG logic!

Reply:
Wow! I had no idea this is possible. Nice  one :)

Reply:
Ah yes good spot! Hmm that's interesting, sounds like a bug in Flax BERT! Feel free to open a PR to fix it, otherwise I can do it after we merge this PR :)